graspologic/cluster/autogmm.py

```diff
@@ -363,6 +371,13 @@ def __init__(
         if max_agglom_size is not None and max_agglom_size < 2:
             raise ValueError("Must use at least 2 points for `max_agglom_size`")
 
+        if n_init is not None:
+            check_scalar(x=n_init, name="n_init", target_type=int, min_val=1)
+
+        run_multiple_init = False
+        if n_init is not None and label_init is None:
+            run_multiple_init = True
```

Comment:
this seems redundant to me. why is this not the same as checking whether n_init > 0 or n_init > 1?

Reply:
It is redundant, I had added it cause I thought it would be convenient notation for whoever reads the code, but can certainly remove it

Reply:
Well, I do see that it's easier to check `run_multiple_init` compare to `n_init` & `label_init` for the if condition later but that's only a difference of 1 line. However, with this variable you have to add more lines defining & checking it (such as here). And I think `n_init>1` implies run multiple init quite clearly.  So overall, I don't think it's worth it.